tests/installation/iso_checksum.pm

```diff
@@ -0,0 +1,31 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Verify data integrity of the iso image by comparing checksum from IBS/OBS with checksum on the worker
```

Comment:
Perhaps you can cut the comment to the width of the license, otherwise it looks too long.

Reply:
Yes, please wrap after 78 or 80 characters